test/unit_tests/problems/constant.hpp

```diff
@@ -0,0 +1,114 @@
+/* -----------------------------------------------------------------------------
+ * Programmer(s): David J. Gardner @ LLNL
+ * -----------------------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2025, Lawrence Livermore National Security,
+ * University of Maryland Baltimore County, and the SUNDIALS contributors.
+ * Copyright (c) 2013-2025, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * Copyright (c) 2002-2013, Lawrence Livermore National Security.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ * -----------------------------------------------------------------------------
+ * Test problem with constant derivatives
+ *
+ *   y' = a
+ *
+ * with the initial condition
+ *
+ *   y(t_0) = y_0
+ *
+ * This problem has analytical solution
+ *
+ *   y(t) = a * (t - t_0) + y_0
```

Comment:
In the file itself, `a=1`:
```suggestion
 *   y' = 1
 *
 * with the initial condition
 *
 *   y(t_0) = y_0
 *
 * This problem has analytical solution
 *
 *   y(t) = (t - t_0) + y_0
```

Reply:
The comments are correct e.g., the RHS function is `N_VScale(SUN_RCONST(1.0), udata->rhs_, ydot);` where `rhs_` is an input to the constructor.